src/rust/engine/process_execution/src/lib.rs

```diff
@@ -252,6 +256,26 @@ pub trait CommandRunner: Send + Sync {
   ) -> Option<ExecuteProcessRequest>;
 }
 
+// TODO possibly move to the MEPR struct, or to the hashing crate?
+pub fn digest(req: MultiPlatformExecuteProcessRequest, metadata: &ExecuteProcessRequestMetadata) -> Digest {
+  let mut hashes: Vec<String> = req
+      .0
+      .values()
+      .map(|ref epr| crate::remote::make_execute_request(epr, metadata.clone()).unwrap())
+      .map(|(_a, _b, er)| er.get_action_digest().get_hash().to_string())
+      .collect();
+  hashes.sort();
```

Comment:
Sorting should be fine here, as one of the contracts of MPEPR is that all EPRs in the request are precisely equivalent.

(I still think we should be using separate digests, rather than merging them, though...)